pkgs/development/python-modules/crossandra/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  result,
+  mypy
+}:
+
+buildPythonPackage rec {
+  pname = "crossandra";
+  version = "2.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "trag1c";
+    repo = "crossandra";
+    rev = "refs/tags/${version}";
+    hash = "sha256-/JhrjXRH7Rs2bUil9HRneBC9wlVYEyfwivjzb+eyRv8=";
+  };
+
+  build-system = [ setuptools mypy ];
```

Comment:
It is very odd that mypy is here. I would rather patch it out of pyproject.toml

Reply:
It uses `mypyc` which is provided by `mypy`

Reply:
never seen that before but seems legit